uni/ulsp/server.icn

```diff
@@ -251,9 +271,51 @@ class Server(
       jrpc_handler.send_msg(msg)
    end
 
+########################################################
+#                textDocument/documentSymbol           #
+########################################################
 
-   method shutdown()
+   method symbolHelp(msg)
+      local filehandler_object, result
+      filehandler_object := openFiles[\msg.get_param_uri()]
+      symbolHandler.setVariables(filehandler_object)
+      if renewSymbols = 1 then {
+         Logger.info("Getting new symbols for " || \msg.get_param_uri())
```

Comment:
If you want to allow this call to always succeed, then 
```
 Logger.info("Getting new symbols for " || (\msg.get_param_uri() | "")
```
However, given that you don't store the result if we don't have a uri, might as well wrap this black with a check if we have uri or not, so it becomes:
```
if \msg.get_param_uri() then }
   Logger.info("Getting new symbols for " || msg.get_param_uri())
   result := symbolHandler.run()
   cacheSymbols[msg.get_param_uri()] := result
}
```
Also, these logs seem like debug msgs rather than info.